prow/github/client.go

```diff
@@ -663,3 +664,27 @@ func (c *Client) FindIssues(query string) ([]Issue, error) {
 	}, &issSearchResult)
 	return issSearchResult.Issues, err
 }
+
+// GetContents uses github repo contents API to retrieve the content of a file.
+// Todo: Support retrieve a directory
+func (c *Client) GetFile(org, repo, filepath string) (string, error) {
+	c.log("GetRemoteFile", org, repo, filepath)
+	var res struct {
+		Object map[string]string `json:"object"`
+	}
+	_, err := c.request(&request{
+		method:    http.MethodGet,
+		path:      fmt.Sprintf("%s/repos/%s/%s/%s", c.base, org, repo, filepath),
```

Comment:
Is this path right? I'm looking at https://developer.github.com/v3/repos/contents/#get-contents and I think you're missing `/contents/`.